prefab/prefab-publishing/mylibrary/build.gradle

```diff
@@ -14,17 +14,15 @@
  * limitations under the License.
  */
 
-plugins {
-    id 'com.android.library'
-}
+apply plugin: 'com.android.library'
```

Comment:
Why the regression? This is the legacy plugin syntax: https://docs.gradle.org/current/userguide/plugins.html#sec:old_plugin_application

Reply:
sample in this repo adopts "apply plugin".

Reply:
That's a reason for those to be fixed, not for this one to regress.